package.json

```diff
@@ -22,5 +22,9 @@
   "homepage": "https://github.com/joomla-projects/joomla-cypress#readme",
   "files": [
     "src"
-  ]
+  ],
+  "devDependencies": {
```

Comment:
Is it really a dev dependency? For this project I would say it is a normal dependency.

Reply:
Since I don't know that much about it, I had the same question. I asked an AI and the shortened answer was:
* Cypress is typically listed as a devDependency rather than a dependency. This is because Cypress is used primarily for testing during the development phase and is not required for the application to run in production ... Since Cypress is a testing framework, it is used to write and execute tests during the development process, making it appropriate to include it under devDependencies.

I doubled checked with search engine and found [stackoverflow](https://stackoverflow.com/questions/61216538/difference-between-dev-dependencies-and-dependencies-specifically-in-cypress-io):
* Cypress is a testing tool and will not be required in a production environment. Therefore, it should be installed as a devDependency as stated in Cypress doc [here](https://docs.cypress.io/guides/getting-started/installing-cypress.html).

I would therefore keep devDependencies.

Reply:
In a project like the CMS it will be definitely a dev dependency, but here not. It is needed to run this project, not only in development mode. here is a different situation.

Reply:
A dev dependency for this project here can be a code style checker or so. But cypress is definitely needed, so it comes as normal dependency.

Reply:
Another example can be found [here](https://github.com/bahmutov/cypress-task-demo/blob/master/package.json#L33).

Reply:
Yes, you're right, I tried devDependencies and it doesn't work. Tested with a new project which uses joomla-cypress in production and therefore sets dependency:joomla-cypress. With devDependencies:cypress-file-upload in joomla-cypress the package is not installed in project with command `NODE_ENV=production npm ci`.

But, the used [cypress-file-upload](https://github.com/abramenal/cypress-file-upload) or [cypress-web-vital](https://github.com/cmorten/cypress-web-vitals) are using peerDependencies. With peerDependencies the tested scenarios with joomla-cypress production project and joomla-cypress development project works too. So once again there are doubts.

But now I committed with 'depencies' to finish and tested extensively.

Tested 'joomla-cypress' usage for empty new project:
* Create new project with old NPM 1.0.3 package behaviour:
  * 'joomla-cypress' in production usage:
    * 'npm init -y && npm install joomla-cypress && rm -rf node_modules && NODE_ENV=production npm ci'
    * -> joomla-cypress is incorrectly present in the node_modules directory
  * 'joomla-cypress' in development only usage:
    * 'npm init -y && npm install joomla-cypress --save-dev'
    * -> cypress-file-upload is missing in the node_modules directory
* Same test with local installed folder 'joomla-cypress' from this commit
  * 'joomla-cypress' in production usage:
    * 'npm init -y && npm install ./joomla-cypress && rm -rf node_modules && NODE_ENV=production npm ci'
    * -> node_modules is correctly empty (only empty '@Colors', '@cypress' and '@types' exist)
  * 'joomla-cypress' in development only usage:
    * 'npm init -y && npm install ./joomla-cypress --save-dev'
    * -> correctly installed are cypress-file-upload, cypress and all depending packages in the node_modules directory

Tested "file:./joomla-cypress" installation with local 5.1-dev joomla-cms, manual-examples and quote_joomla.
  * Only one Cypress is installed and test suites are running successfully